src/actions/fetchSinglePostAction.js

```diff
@@ -20,6 +21,19 @@ export const fetchSuccess = (post) => ({
   payload: post,
 });
 
+const fetchRelatedPosts = (category, singlePostSlug) => server.get(`feed?offset=0&limit=3&category=${category}`)
```

Comment:
@dusmel I think we should create a constant to hold the limit for `related posts` and increase this one a little bit more.

Reply:
So the post itself can't go beyond 55 characters that mean is very short, adding more cards on the right side will make the whole design looks weird